test/event_test.py

```diff
@@ -834,6 +870,57 @@ def test_poll(self):
         self.assertEqual(pygame.event.poll().type, pygame.NOEVENT)
 
 
+class EventModuleTestsWithTiming(unittest.TestCase):
+    __tags__ = ["timing"]
+
+    def test_event_wait(self):
```

Comment:
This test fails when run locally on windows:

```
======================================================================
ERROR: test_event_wait (__main__.EventModuleTestsWithTiming)
Ensure wait() waits for an event on the queue.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\dan\Programming\pygame\test\event_test.py", line 880, in test_event_wait
    pygame.event.post(event)
pygame.error: video system not initialized

----------------------------------------------------------------------
```

I assume it needs a `pygame.display.init()` in the setup somewhere.

Reply:
Oh yeah, good catch! I had tested this function before I moved it into its own class and it worked then, but I missed `setUp` and `tearDown` functions, which I've added now!